protoc-gen-kotlin/lib/src/commonTest/kotlin/CodeGeneratorTest.kt

```diff
@@ -0,0 +1,143 @@
+package pbandk
```

Comment:
This file should be in a directory that matches its package name. I.e. `src/commonTest/kotlin/pbandk` instead of `src/commonTest/kotlin`. Though since the rest of the code under `protoc-gen-kotlin` is in the `pbandk.gen` package, the test file should really be in that package too.